kubernetes-upstream.Jenkinsfile

```diff
@@ -74,6 +74,15 @@ pipeline {
             steps {
                 sh 'cd ${TESTDIR}; vagrant up k8s1-${K8S_VERSION}'
                 sh 'cd ${TESTDIR}; vagrant up k8s2-${K8S_VERSION}'
+            }
+        }
+
+        stage('BDD-tests'){
```

Comment:
Thanks for adding this, hopefully this will help with the provisioning timeout problems on this job.